examples/connext_dds/network_capture/example0-shared_memory_and_udp/.gitignore

```diff
@@ -0,0 +1,17 @@
+# Solution and project files are created by rtidddsgen
```

Comment:
I was wondering if the "example0" adds a lot here. Maybe you could simply go with shared_memory_and_udp"